### PR TITLE
docs(contributing.md): update CONTRIBUTING.md with latest info

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,92 +64,60 @@ Guidelines for bug reports:
 
 3. Isolate the problem &mdash; **create a [reduced test case](https://css-tricks.com/reduced-test-cases/)** with a live example. You can possibly use [this JSBin example](http://jsbin.com/axedog/7/edit) as a starting point.
```
Comment (mister-ben) [inline CONTRIBUTING.md]: This jsbin has video.js 4.1

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -124,152 +126,199 @@ Assign the original repo to a remote called "upstream"
...
+#### Sandbox test directory
 
-Start a new development branch
+There's a sandbox directory where you can add any file and it won't get tracked in git. To start you can copy the example index file and see a working version of a player (using the local source code) by loading it in a browser.
 
-```bash
-contrib feature start
+```sh
+$ cp sandbox/index.html.example sandbox/index.html
+$ open sandbox/index.html
```
Comment (mister-ben) [inline CONTRIBUTING.md]: Is it a good idea to encourage opening from file:// ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,23 +1,57 @@
...
+## Filing issues
 
-Filing issues
--------------
 [GitHub Issues](https://github.com/videojs/video.js/issues) are used for all discussions around the codebase, including **bugs**, **features**, and other **enhancements**.
 
+When filling out an issue, make sure to fill out the questions in the 
 ### Reporting a Bug
 
 **A bug is a demonstrable problem** that is caused by the code in the repository. Good bug reports are extremely helpful. Thank You!
```
Comment (mister-ben) [inline CONTRIBUTING.md]: Maybe add a more explicit note that bugs in plugins should be filed against those plugins.

File: CONTRIBUTING.md (hunk 2 above)
Comment (misteroneill) [inline CONTRIBUTING.md]: Maybe suggest running `npm start` and heading over to `localhost:9999/sandbox/`?

Comment (gkatsev) [inline CONTRIBUTING.md]: Yes, though, that's literally the next section.

File: CONTRIBUTING.md (hunk 3 above)
Comment (gkatsev) [inline CONTRIBUTING.md]: Good idea.

Comment (gkatsev) [inline CONTRIBUTING.md]: Done.

**Pull request merged**